pages/common/gnmic.md

```diff
@@ -0,0 +1,24 @@
+# gnmic
+
+> A gNMI CLI client that provides full support for Capabilities, Get, Set and Subscribe RPCs with collector capabilities.
+> More information: <https://gnmic.kmrd.dev>.
+
+- Request capabilities (gnmic capabilities):
+
+`gnmic --address {{ip:port}} --insecure capabilities`
+
+- Username and password can be given to any command:
+
+`gnmic --address {{ip:port}} --username {{username}} --password {{password}} --insecure capabilities`
+
+- Get request (gnmic get):
+
+`gnmic -a {{ip:port}} --insecure get --path {{path}}`
+
+- Set request (gnmic set):
```

Comment:
Going off if the [gNMI specification section 3.4](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#34-modifying-state)

```suggestion
- Update device state at {{path}}:
```